composer.json

```diff
@@ -19,11 +19,11 @@
         "guzzlehttp/guzzle": "^6.5 || ^7.0",
         "ext-json": "*",
         "art4/json-api-client": "^1.0",
-        "predis/predis": "^1.0"
+        "predis/predis": "^1.0",
+        "php-di/php-di": "^6.4"
```

Comment:
what is it? Can u please explain what you are doing here and why it should be in require instead of require-dev

Reply:
Additionally, you have updated the minimum compatible version. It's not right if you're not using features from 6.4